lib/tasks/tests.rake

```diff
@@ -17,7 +17,7 @@ task :default do
 end
 
 Rake::TestTask.new('test:unit') do |t|
-  t.pattern = 'test/{controllers,lib,models,interactors}/**/*_test.rb'
+  t.pattern = 'test/{controllers,jobs,lib,mailers,models,services}/**/*_test.rb'
```

Comment:
Would it be easier and more future-proof to just run all tests in all dirs, except the integration dir?